services/account-without-juggler/index.ts

```diff
@@ -0,0 +1,39 @@
+import { Application, Server } from 'loopback-next/packages/core';
+import { AccountController } from './controllers/AccountController';
+import { accountRepository } from './repositories/account';
+
+class AccountMicroservice extends Application {
+  private _startTime: Date;
+
+  constructor() {
+    super();
+    const app = this;
+    app.controller(AccountController);
+    app.bind('repositories.account').toClass(accountRepository);
```

Comment:
I think we can land and create an issue (feature) for the sugar API mentioned by @bajtos. This PR is huge already.